src/main/java/io/pinecone/unsigned_indices_model/QueryResponseWithUnsignedIndices.java

```diff
@@ -8,13 +8,43 @@
 import java.util.Collections;
 import java.util.List;
 
+/**
+ * This class represents the response to a query, where the `ScoredVector` objects in the `matches` list
+ * contain `SparseValuesWithUnsignedIndices` instead of the standard `SparseValues`.
+ * <p>
+ * The `QueryResponseWithUnsignedIndices` class contains the following fields:
+ * - `matches`: a list of `ScoredVectorWithUnsignedIndices` objects, representing the matching vectors
+ * - `namespace`: the namespace of the query
+ * - `usage`: the usage information for the query
+ * <p>
+ * The class provides a constructor that takes a `QueryResponse` object and converts it to a
+ * `QueryResponseWithUnsignedIndices` object, as well as getter and setter methods for each of the fields.
+ * It also includes a method to convert a list of `ScoredVector` objects to a list of
+ * `ScoredVectorWithUnsignedIndices` objects.
+ */
```

Comment:
This might be convention, so ignore me if it is, but this seems like too much info for a docstring. I think you can just define the class at a high level and define the params. Plus, outlining each method like this in a docstring means it'll be a pain to update when we add new ones/remove old ones.

Reply:
Yeah but these are custom classes which are used for converting from unsigned 32-bit integers to signed 32-bit integers and vice-versa. There are also a correspond set of classes like ScoredVector, QueryResponse, SparseValues, and Vector generated using protoc command which represents sparse indices in signed 32 bit format.